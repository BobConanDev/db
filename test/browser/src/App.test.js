import { render, screen } from '@testing-library/react';
import App from './App';

test('lists flureedb fns', () => {
  const {asFragment} = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
