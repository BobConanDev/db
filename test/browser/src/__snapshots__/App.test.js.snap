// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lists flureedb fns 1`] = `
<DocumentFragment>
  <ul
    class="App"
  >
    <li>
      authenticate
    </li>
    <li>
      block_query
    </li>
    <li>
      block_range
    </li>
    <li>
      block_range_with_txn
    </li>
    <li>
      close
    </li>
    <li>
      close_listener
    </li>
    <li>
      connect
    </li>
    <li>
      connect_p
    </li>
    <li>
      db
    </li>
    <li>
      db_schema
    </li>
    <li>
      delete_ledger
    </li>
    <li>
      history_query
    </li>
    <li>
      http_signature
    </li>
    <li>
      jldCommit
    </li>
    <li>
      jldConnect
    </li>
    <li>
      jldCreate
    </li>
    <li>
      jldDb
    </li>
    <li>
      jldLoad
    </li>
    <li>
      jldQuery
    </li>
    <li>
      jldStage
    </li>
    <li>
      jldStatus
    </li>
    <li>
      ledger_stats
    </li>
    <li>
      listen
    </li>
    <li>
      listeners
    </li>
    <li>
      monitor_tx
    </li>
    <li>
      multi_query
    </li>
    <li>
      new_ledger
    </li>
    <li>
      password_generate
    </li>
    <li>
      password_login
    </li>
    <li>
      query
    </li>
    <li>
      renew_token
    </li>
    <li>
      set_logging
    </li>
    <li>
      signed_query
    </li>
    <li>
      transact
    </li>
  </ul>
</DocumentFragment>
`;

exports[`lists flureedb fns: <></> 1`] = `
<DocumentFragment>
  <ul
    class="App"
  >
    <li>
      authenticate
    </li>
    <li>
      block_query
    </li>
    <li>
      block_range
    </li>
    <li>
      block_range_with_txn
    </li>
    <li>
      close
    </li>
    <li>
      close_listener
    </li>
    <li>
      connect
    </li>
    <li>
      connect_p
    </li>
    <li>
      db
    </li>
    <li>
      db_schema
    </li>
    <li>
      delete_ledger
    </li>
    <li>
      history_query
    </li>
    <li>
      http_signature
    </li>
    <li>
      jldCommit
    </li>
    <li>
      jldConnect
    </li>
    <li>
      jldCreate
    </li>
    <li>
      jldDb
    </li>
    <li>
      jldLoad
    </li>
    <li>
      jldQuery
    </li>
    <li>
      jldStage
    </li>
    <li>
      jldStatus
    </li>
    <li>
      ledger_stats
    </li>
    <li>
      listen
    </li>
    <li>
      listeners
    </li>
    <li>
      monitor_tx
    </li>
    <li>
      multi_query
    </li>
    <li>
      new_ledger
    </li>
    <li>
      password_generate
    </li>
    <li>
      password_login
    </li>
    <li>
      query
    </li>
    <li>
      renew_token
    </li>
    <li>
      set_logging
    </li>
    <li>
      signed_query
    </li>
    <li>
      transact
    </li>
  </ul>
</DocumentFragment>
`;
